File: Code Chef/JavaScript/600-699/643 Hackerman PRIMEDICE.js
let input = '';
process.stdin.on('data', (data) => input += data);

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => strItem.trim().split(' ').map(Number));
    
    return intArray;
};

const findWhoWon = (numberOfAlice, numberOfBob) => {
    const sum = numberOfAlice + numberOfBob;
    
    (sum === 2 || sum === 3 || sum === 5 || sum === 7 || sum === 11)
        ? console.log('Alice')
        : console.log('Bob');
};

process.stdin.on('end', () => {
    const testcases = strToInt(input.trim().split('\n'));
    
    testcases.forEach((testcase) => {
        const [numberOfAlice, numberOfBob] = testcase;
        findWhoWon(numberOfAlice, numberOfBob);
    });
});

File: Code Chef/C Programming/600-699/643 Hackerman PRIMEDICE.c
#include <stdio.h>
int main(void)
{
	int t,a,b;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
	    scanf("%d %d",&a,&b);
	    if((a+b)==2||(a+b)==3||(a+b)==5||(a+b)==7||(a+b)==11)
	    {
	        printf("Alice\n");
	    }
	    else
	    {
	        printf("Bob\n");
	    }
	}
	return 0;
}